tests/logging/test_logger.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "test_logger.h"
+
+#include <aws/common/byte_buf.h>
+
+#include <stdarg.h>
+#include <stdio.h>
+
+#define TEST_LOGGER_MAX_LOG_LINE_SIZE 256
+
+int s_test_logger_log_fn(struct aws_logger *logger, enum aws_log_level log_level, aws_log_subject_t subject, const char *format, ...) {
+    va_list format_args;
+    va_start(format_args, format);
+
+    static char buffer[TEST_LOGGER_MAX_LOG_LINE_SIZE];
+
+#ifdef WIN32
+    int written = vsnprintf_s(buffer, MAX_LOG_LINE_SIZE, _TRUNCATE, format, format_args);
```

Comment:
Shouldn't this be TEST_LOGGER_MAX_LOG_LINE_SIZE?

Reply:
Yes it should.